aws-android-sdk-apigateway-core/build.gradle

```diff
@@ -9,13 +9,6 @@ android {
         versionCode 1
         versionName '1.0'
     }
-
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-        }
-    }
```

Comment:
Removed these declarations everywhere since the minification is disabled everywhere. Also the purpose of this is to minify the shipped artifact.